Services/VIWIS/interfaces/interface.generalDBIndependentFeedback.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * We may not rely on ILIAS-Feedbacks, since they seem instanctly
+ * to store any information in Database. We will create our own simple
+ * feedback-container.
+ */
+
+interface generalDBIndependentFeedback {
```

Comment:
Ich möchte keine feedbacks in die datanbank schreiben, was die ilias-eigenen feedbacks einfach so tun,also habe ich ein interface geschrieben, welches ein feedback speichert und auf abruf wiedergibt.